src/wcs1_0/src/test/java/org/geoserver/wcs/test/WCSTestSupport.java

```diff
@@ -140,4 +140,18 @@ protected void setupRasterDimension(
         info.getMetadata().put(metadata, di);
         getCatalog().save(info);
     }
+
+    protected void setupFixedValueRasterDimension(
+            QName layer, String metadata, DimensionPresentation presentation, Double resolution) {
```

Comment:
add a parameter for the fixed values string so that one can test with different method and using time or elevation. Moreover more test are needed and possibly using more then one value.